src/main/java/org/apache/rocketmq/dashboard/service/impl/ConsumerServiceImpl.java

```diff
@@ -312,14 +320,16 @@ public boolean deleteSubGroup(DeleteSubGroupRequest deleteSubGroupRequest) {
         return true;
     }
 
-    private void deleteResources(String topic, String brokerName, ClusterInfo clusterInfo) throws Exception {
+    private void deleteResources(String topic, String brokerName, ClusterInfo clusterInfo, boolean deleteInNsFlag) throws Exception {
         mqAdminExt.deleteTopicInBroker(Sets.newHashSet(clusterInfo.getBrokerAddrTable().get(brokerName).selectBrokerAddr()), topic);
         Set<String> nameServerSet = null;
         if (StringUtils.isNotBlank(configure.getNamesrvAddr())) {
             String[] ns = configure.getNamesrvAddr().split(";");
             nameServerSet = new HashSet<>(Arrays.asList(ns));
         }
-        mqAdminExt.deleteTopicInNameServer(nameServerSet, topic);
+        if (deleteInNsFlag) {
```

Comment:
Any unit test coverage for this logic?

Reply:
Yes, these lines of code is covered . The reason for the decrease in coverage this time is that the code modified by this patch(#59) is not covered. Since it is a different branch, the single test cannot be directly added this time, and it will be added in the subsequent submit
![image](https://user-images.githubusercontent.com/18254437/148146758-6a236857-6019-4891-94e0-abb58b669940.png)

Reply:
Got it